docker/config.sh

```diff
@@ -16,12 +17,25 @@ function install_mc {
         && mv ./mc /usr/bin/ 
 }
 
-function install_gdal_python_reqs {
-    apt-get install -y libgdal-dev \
-        && apt-get -y install --reinstall build-essential
+function install_gdal {
+    wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
+    sudo apt-get install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
+    sudo apt-get update
+
+    sudo apt-get install -y -V libproj-dev libgeos-dev libarrow-dev libparquet-dev cmake
```

Comment:
to confirm, is this to install arrow/parquet things because _we_ wanna use GDAL + parquet or because GDAL 3.8.2 requires these?

Reply:
The GDAL parquet driver requires them

Reply:
for the other packages here, proj and geos are needed for gdal, cmake is used to run the build/install